src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -1189,6 +1187,17 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     }, DEFAULT_COLLAB_DEBOUNCE);
 
+    /**
+     * Returns detailsSidebarProps replacing missing props with their defaults
+     *
+     * @private
+     * @return {DetailsSidebarProps} the merged detailsSidebarProps
+     */
+    getdetailsSidebarProps(): DetailsSidebarProps {
+        const { detailsSidebarProps } = this.props;
+        return extend(ContentSidebar.defaultProps.detailsSidebarProps, detailsSidebarProps);
```

Comment:
This will allow us to nest further if we decide to put objects in detailsSidebarProps

Reply:
Should the `DetailsSidebar` already have these default props? Are there any cases where the DetailsSidebar is used outside of the ContentSidebar?

Reply:
As far as I know, the tabs we're building (details, AF, skills etc.) Will always be used in the content sidebar